agent/__init__.py

```diff
@@ -480,6 +480,20 @@ def setup_endpoints(dev, debug):
 
 
 def fetch_credentials(debug, dev):
+
+    def clear_credentials(path):
```

Comment:
@a-martynovich Take a close look at this function and imagine how this could be exploited to remove arbitrary files.

Reply:
yes...about a glob i forgot. thank you.